dali/operators/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ set_target_properties(${dali_operator_lib} PROPERTIES
     LIBRARY_OUTPUT_DIRECTORY "${DALI_LIBRARY_OUTPUT_DIR}")
 target_link_libraries(${dali_operator_lib} PUBLIC ${dali_lib} ${dali_kernel_lib} ${dali_core_lib})
 target_link_libraries(${dali_operator_lib} PRIVATE dynlink_cuda ${DALI_LIBS})
+target_link_libraries(${dali_operator_lib} INTERFACE sndfile)
```

Comment:
what does this mean? Can't it be PRIVATE? It's not part of the interface, is it?

Reply:
Now, that we moved things to .cc it can be private. Changes